src/Optimizely/Config/DatafileProjectConfig.php

```diff
@@ -230,19 +273,19 @@ public function __construct($datafile, $logger, $errorHandler)
 
         $this->_accountId = $config['accountId'];
         $this->_projectId = $config['projectId'];
-        $this->_anonymizeIP = isset($config['anonymizeIP'])? $config['anonymizeIP'] : false;
-        $this->_botFiltering = isset($config['botFiltering'])? $config['botFiltering'] : null;
+        $this->attributes = $config['attributes'] ?: [];
+        $this->audiences = $config['audiences'] ?: [];
+        $this->events = $config['events'] ?: [];
+        $this->typedAudiences = isset($config['typedAudiences']) ? $config['typedAudiences'] : [];
+        $this->_anonymizeIP = isset($config['anonymizeIP']) ? $config['anonymizeIP'] : false;
+        $this->_botFiltering = isset($config['botFiltering']) ? $config['botFiltering'] : null;
         $this->_revision = $config['revision'];
         $this->_sendFlagDecisions = isset($config['sendFlagDecisions']) ? $config['sendFlagDecisions'] : false;
 
         $groups = $config['groups'] ?: [];
         $experiments = $config['experiments'] ?: [];
-        $events = $config['events'] ?: [];
-        $attributes = $config['attributes'] ?: [];
-        $audiences = $config['audiences'] ?: [];
-        $typedAudiences = isset($config['typedAudiences']) ? $config['typedAudiences']: [];
         $rollouts = isset($config['rollouts']) ? $config['rollouts'] : [];
-        $featureFlags = isset($config['featureFlags']) ? $config['featureFlags']: [];
+        $featureFlags = isset($config['featureFlags']) ? $config['featureFlags'] : [];
```

Comment:
actually i started my before the pr of duplicate exp key issue was merged with master so i just update my branch with the latest master branch to use that instead of doing that task again